src/js/edu-benefits/components/EducationView.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { formatReviewDate } from '../../common/schemaform/helpers';
+
+export default function EducationView({ formData }) {
+  let from = '';
+  let to = '';
+  if (formData.dateRange) {
+    from = formatReviewDate(formData.dateRange.from);
+    to = formatReviewDate(formData.dateRange.to);
+  }
+
+  return (
+    <div>
```

Comment:
This is what we've normally been doing, so I think this is good. Alex can create an issue to change it later if we need to.